build.sbt

```diff
@@ -7,14 +7,27 @@ lazy val gcTask = gc := {
   System gc()
 }
 
+ThisBuild / githubWorkflowBuild := Seq(
+  WorkflowStep.Sbt(List("mimaReportBinaryIssues")),
+  WorkflowStep.Sbt(List("clean")),
+  WorkflowStep.Sbt(List("coverage")),
+  WorkflowStep.Sbt(List("test"))
+)
+
+ThisBuild / githubWorkflowBuildPostamble ++= Seq(
+  WorkflowStep.Sbt(List("coverageReport")),
+  WorkflowStep.Sbt(List("coveralls"))
+)
+
+ThisBuild / scalaVersion := "2.12.14"
+ThisBuild / crossScalaVersions := Seq("2.10.7", "2.11.12", scalaVersion.value, "2.13.6")
```

Comment:
We have to pull both `scalaVersion` and `crossScalaVersions` out of the `settings` in `root` otherwise sbt-github-actions can't detect the setting when generating github actions.